reana-ui/src/pages/workflowList/WorkflowList.js

```diff
@@ -111,6 +115,10 @@ function Workflows() {
     interval.current = null;
   };
 
+  if ("next_url" in queryPrams) {
+    return <Redirect to={queryPrams.next_url} />;
+  }
+
```

Comment:
What do you think about moving the redirection logic somewhere else?

Two possible options:
1. Move it to `WorkflowListPage`, so that we do not add more complexity to `Workflows`.
2. Create a new route and modify reana-server's configuration so that, after logging in, the user is redirected there. We can achieve this by modifying [these options](https://github.com/inveniosoftware/invenio-oauthclient/blob/v1.5.4/invenio_oauthclient/contrib/cern_openid.py#L148-L151) of `invenio-oauthclient`.

We should also handle login errors, for example by showing some notifications to the user, but this can be done in another PR.